ci/azure/posix.yml

```diff
@@ -73,24 +73,42 @@ jobs:
 
   steps:
     - script: |
-        if [ "$(uname)" == "Linux" ]; then
-          sudo apt-get update
-          sudo apt-get install -y libc6-dev-i386 $EXTRA_APT
-        fi
+        sudo apt-get update
+        sudo apt-get install -y libc6-dev-i386 $EXTRA_APT
       displayName: 'Install extra packages'
+      condition: eq('${{ parameters.name }}', 'Linux')
 
-    - script: echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
-      displayName: 'Set conda path'
+    - bash: echo '##vso[task.prependpath]$(HOME)/miniconda3/bin'
+      displayName: 'Add conda to PATH'
 
-    - script: ci/setup_env.sh
-      displayName: 'Setup environment and build pandas'
+    - bash: sudo chown -R $USER $CONDA
+      displayName: Take ownership of conda installation
+      condition: eq('${{ parameters.name }}', 'macOS')
 
-    - script: |
-        source activate pandas-dev
+    - bash: conda update --quiet --yes conda
+      displayName: Update conda
+
+    - bash: conda env create --quiet --file $(ENV_FILE)
+      displayName: Create Anaconda environment
+
+    - bash: |
+        eval "$(conda shell.bash hook)"
+        conda activate pandas-dev
```

Comment:
Not sure why `source activate pandas-dev` doesn't work here.

Reply:
source activate is the old way, conda activate is the new (both worked for a long time, but conda activate is the way)